pkg/cmd/flags/logger.go

```diff
@@ -21,6 +21,7 @@ Possible options:
   --log warn                     | warning log level
   --log error                    | error log level
   --log panic                    | panic log level
+  --log file:/path/to/file       | write the logs to a specified file. create/trim the file if exists (default: stderr)
```

Comment:
Other options are mutually exclusive, this one is not. Somehow the user should be told that.

Reply:
Makes sense, I considered it, but as `aggregate` is also like `file` and can be used with other flags, I didn't mind. Changing to what Yaniv suggested.